src/app.js

```diff
@@ -15,9 +15,9 @@ const app = express()
 
 // Use HB before all other app middleware.
 Honeybadger.configure({
-  apiKey: process.env.HONEYBADGER_API_KEY
+  apiKey: process.env.HONEYBADGER_API_KEY,
+  environment: process.env.HONEYBADGER_ENV
 })
-app.use(Honeybadger.requestHandler)
```

Comment:
this removes the handler that alerted HB automatically at all times - see https://www.honeybadger.io/blog/honeybadger-for-node-js-is-now-awesome/